### PR TITLE
Relayer: Remove incorrect comments, change uint32->Gas, SetDefaultRelayProvider->SetDefaultDeliveryProvider

### PR DESCRIPTION
**Pull request by derpy-duck**
also: Revert in WormholeRelayerDelivery if relayer refund address is not payable

### CODE REVIEW

File: ethereum/contracts/relayer/wormholeRelayer/WormholeRelayerDelivery.sol [hunk 1]
```diff
@@ -274,17 +282,14 @@ abstract contract WormholeRelayerDelivery is WormholeRelayerBase, IWormholeRelay
...
-            //decode returned Cancelled error
-
+            //decode returned error if it is one of three known types
             bool knownError;
-            uint32 gasUsed_;
+            Gas gasUsed_;
             (gasUsed_, knownError) = tryDecodeExecuteInstructionError(revertData);
             results = DeliveryResults(
-                knownError? Gas.wrap(gasUsed_) : vaaInfo.gasLimit,
+                knownError? gasUsed_ : vaaInfo.gasLimit,
```
Comment (nonergodic) [inline ethereum/contracts/relayer/wormholeRelayer/WormholeRelayerDelivery.sol:292]: missing space between `knownError` and `?` operator

**Review by nonergodic — COMMENTED** [review]
Only reviewed the Solidity parts

File: ethereum/contracts/relayer/wormholeRelayer/WormholeRelayerGovernance.sol [hunk 2]
```diff
@@ -38,7 +38,7 @@ abstract contract WormholeRelayerGovernance is WormholeRelayerBase, ERC1967Upgra
      *   - https://github.com/wormhole-foundation/wormhole/blob/main/whitepapers/0002_governance_messaging.md
      */
 
-    //Right shifted ascii encoding of "WormholeRelayer"
+    //Right shifted ascii encoding of "CoreRelayer"
     bytes32 private constant module =
         0x000000000000000000000000000000000000000000436f726552656c61796572;
```
[the remark this answers is not in the document]
Comment (derpy-duck) [reply ethereum/contracts/relayer/wormholeRelayer/WormholeRelayerGovernance.sol:43]: Yes! will address seperately

File: ethereum/contracts/relayer/wormholeRelayer/WormholeRelayerGovernance.sol [hunk 3]
```diff
@@ -48,29 +48,29 @@ abstract contract WormholeRelayerGovernance is WormholeRelayerBase, ERC1967Upgra
...
-     *   - bytes32 newImplementation - bytes32 instead of address for general consistency reasons
+     *   - bytes32 newImplementation
      */
     uint8 private constant GOVERNANCE_ACTION_CONTRACT_UPGRADE = 2;
 
     /**
      * Sets the default relay provider for the WormholeRelayer. Has no equivalent in the core bridge.
      *
      * Action Parameters:
-     *   - bytes32 newProvider - bytes32 instead of address for general consistency reasons
```
[the remark this answers is not in the document]
Comment (derpy-duck) [reply ethereum/contracts/relayer/wormholeRelayer/WormholeRelayerGovernance.sol:73]: agreed. Will reintroduce in the future comment pass

**Pull request merged**